index.js

```diff
@@ -31,4 +31,36 @@ module.exports = {
     'no-val': require('./rules/no-val'),
     'no-wrap': require('./rules/no-wrap')
   }
+  configs: {
+    deprecated: {
+      "jquery/no-ajax": 2,
```

Comment:
Hmm, I dunno if these should be jquery/ prefixed or not. Cause we're in the plugin itself. Not sure...